Makefile

```diff
@@ -1,4 +1,4 @@
-PACKAGES := $(shell glide novendor)
+PACKAGES := $(shell glide novendor | grep -v \/testdata\/)
```

Comment:
nit: `grep -v '/testdata/'` is more readable.

Reply:
+1, will push a commit